app/views/layouts/_notification_box.html.haml

```diff
@@ -0,0 +1,16 @@
+.box.box-info
+  .box-header.with-border
+    %h3.box-title= t('Latest Notifications')
+    .box-tools.pull-right
+      %button.btn.btn-box-tool{ 'data-widget' => 'collapse', type: 'button' }
+        %i.fa.fa-minus
+      %button.btn.btn-box-tool{ 'data-widget' => 'remove', type: 'button' }
+        %i.fa.fa-times
+  .box-body
+    %table.table.no-margin
+      %thead
+        %tr
+          %th
+      %tbody
```

Comment:
Add please class 'messages' to 'tbody'. This will add dynamically ActionCable  Notifications.
`%tbody.messages`